src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncDisposableTests.cs

```diff
@@ -155,17 +184,17 @@ class C : System.IAsyncDisposable
         };
         await x();
     }
-    public async Task DisposeAsync()
+    public async ValueTask DisposeAsync()
     {
         System.Console.Write(""DisposeAsync1 "");
         await Task.Delay(10);
         System.Console.Write(""DisposeAsync2 "");
     }
 }
 ";
-            var comp = CreateCompilationWithMscorlib46(source + s_interfaces, options: TestOptions.DebugExe);
+            var comp = CreateCompilationWithTasksExtensions(source + s_interfaces, options: TestOptions.DebugExe);
             comp.VerifyDiagnostics();
-            CompileAndVerify(comp, expectedOutput: "C body DisposeAsync1 DisposeAsync2 end");
+            CompileAndVerify(comp, expectedOutput: "C body DisposeAsync1 DisposeAsync2 end", verify: Verification.Skipped);
```

Comment:
nit: unless testing for warnings, `CompileAndVerify` already checks for errors, so no need for `VerifyDiagnostics`